scripts/nuget_publish.sh

```diff
@@ -14,8 +14,8 @@ XCODEBUILD_WRAPPER_LOCATION='scripts/xcodebuild_wrapper.sh'
 
 # Extra arguments for xcode build to mimic nuget publishing environment  
 DERIVED_DATA_EXTRA_ARG='-derivedDataPath DerivedData'
-IOS_XCCONFIG_EXTRA_ARG='-xcconfig .ado/xcconfig/publish_overrides_ios.xcconfig'
-MACOS_XCCONFIG_EXTRA_ARG='-xcconfig .ado/xcconfig/publish_overrides_macos.xcconfig'
+STANDARD_XCCONFIG_EXTRA_ARG='-xcconfig .ado/xcconfig/publish_overrides.xcconfig'
```

Comment:
>publish_overrides [](start = 53, length = 17)

what does this file look like?  $(ARCHS_STANDARD)?

Reply:
It doesn't override the Xcode default of `ARCHS=$(ARCHS_STANDARD)`